### PR TITLE
Camel Quarkus 2.16.0 pre-release tasks

### PR DESCRIPTION
**Pull request by aldettinger**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -49,27 +49,27 @@
...
+        <quarkus-qpid-jms.version>0.41.0</quarkus-qpid-jms.version><!-- https://repo1.maven.org/maven2/org/amqphub/quarkus/quarkus-qpid-jms-bom/ -->
 
         <!-- Compile dependency versions (keep sorted alphabetically) -->
         <!-- Items annotated with @sync can be updated by running mvn cq:sync-versions -N -e -->
         <ahc.version>2.12.3</ahc.version><!-- @sync com.cloudbees.thirdparty:zendesk-java-client:${zendesk-client-version} dep:org.asynchttpclient:async-http-client -->
         <animal-sniffer.version>1.18</animal-sniffer.version>
         <antlr3.version>3.5.2</antlr3.version><!-- Spark, Stringtemplate and probably others -->
         <avro.version>1.11.1</avro.version><!-- @sync io.quarkus:quarkus-bom:${quarkus.version} dep:org.apache.avro:avro -->
-        <awssdk.version>2.17.291</awssdk.version><!-- @sync io.quarkiverse.amazonservices:quarkus-amazon-services-parent:${quarkiverse-amazonservices.version} prop:awssdk.version -->
+        <awssdk.version>2.19.9</awssdk.version><!-- @sync io.quarkiverse.amazonservices:quarkus-amazon-services-parent:${quarkiverse-amazonservices.version} prop:awssdk.version -->
```
Comment (jamesnetherton) [inline pom.xml:72]: Should this be put back to `2.17.291` given that the `quarkiverse-amazonservices.version` update has been abandoned?

Comment (aldettinger) [reply pom.xml:72]: Indeed, that's strange because I should have regenerated everything locally.
How would you regenerate the boms ?

Comment (jamesnetherton) [reply pom.xml:72]: > How would you regenerate the boms ?

I would have done this to resync properties after modifying `quarkiverse-amazonservices.version`:

```
mvn cq:sync-versions -N -e
```

Then run `clean install` from the `poms` module until the BOM generation step had completed.

Comment (aldettinger) [reply pom.xml]: Indeed, I probably messed up something. The revert should be effective now. Thanks James :+1:

**Review by zbendhiba — APPROVED** [review]

**Pull request merged**